frontend/public/components/catalog/catalog-page.tsx

```diff
@@ -104,16 +104,23 @@ export class CatalogListPage extends React.Component<CatalogListPageProps, Catal
       projectTemplateItems = this.normalizeTemplates(projectTemplateMetadata);
     }
 
-    return _.sortBy(
-      [
-        ...clusterServiceClassItems,
-        ...imageStreamItems,
-        ...templateItems,
-        ...extensionItems,
-        ...projectTemplateItems,
-      ],
-      'tileName',
-    );
+    const items = [
+      ...clusterServiceClassItems,
+      ...imageStreamItems,
+      ...templateItems,
+      ...extensionItems,
+      ...projectTemplateItems,
+    ];
+
+    const filteredItems = _.filter(items, function(item) {
```

Comment:
Change the callback to an arrow function.

Reply:
```suggestion
    const filteredItems = _.filter(items, ['obj', 'metadata', 'annotations', 'apps.kubernetes.io/internal-object']);
```

Reply:
Updated